lib/mayor_game_web/templates/dashboard/show.html.heex

```diff
@@ -114,6 +125,33 @@
           <% end %>
         </div>
       <% end %>
+      
+      <!-- sort options-->
+      <div class="flex flex-row items-center space-x-1 pb-6">
+        <div class="flex-1" />
+        <button
```

Comment:
Same question here — this is a great idea, but where is `sort_by_name` instantiated?

Reply:
Same as above, they were wiped by a recent merge commit.